hack/envtest/_matrix/v1.24.2.yaml

```
@@ -0,0 +1,2 @@
+go: 1.18
```

```suggestion
go: 1.16
```

According to https://github.com/kubernetes/kubernetes/blob/v1.24.2/go.mod#L9